### PR TITLE
Preparation for the first release

### LINKED ISSUE #5: Update README.md
Needs to be more like the Akka or Akka HTTP one, short explanation what it is, links to chats and docs etc.

### PR DESCRIPTION
**Pull request by 2m**
And fixes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+
+This repository contains interfaces to interact with an Akka Cluster.
+
+Documentation
+-------------
+
+See [reference](http://developer.lightbend.com/docs/akka-cluster-management/0.1/) and [API](http://developer.lightbend.com/docs/api/akka-cluster-management/0.3/akka/cluster/http/management/index.html) documentation pages.
+
+Contributions & Maintainers
+---------------------------
```
Comment (ktoso) [inline README.md]: I guess that's true? Not sure hm

File: README.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+*This project does not have contributors, it only has maintainers—frequent and infrequent—and everyone helps out.*
+We love new maintainers as well as old maintainers. :-)
+The Akka core team keeps an eye on the project to assure its overall coherence but does not fully support these modules.
+
+Contributions are very welcome, see [CONTRIBUTING.md](https://github.com/akka/akka-cluster-management/blob/master/CONTRIBUTING.md) or skim [existing tickets](https://github.com/akka/akka-cluster-management/issues) to see where you could help out.
+
+Caveat Emptor
+-------------
+
+A component in this project does not have to obey the rule of staying binary compatible between releases. Breaking API changes may be introduced without notice as we refine and simplify based on your feedback. A module may be dropped in any release without prior deprecation. The Lightbend subscription does not cover support for these modules.
```
Comment (ktoso) [inline README.md]: For lack of better wording OK to keep as is now 👍

File: project/Common.scala [hunk 3]
```diff
@@ -26,8 +26,6 @@ object Common extends AutoPlugin {
 
     licenses := Seq(("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0"))),
 
-    scalaVersion := crossScalaVersions.value.head,
-    crossScalaVersions := Dependencies.ScalaVersions,
```
Comment (patriknw) [inline project/Common.scala]: um, suddenly travis becomes part of the build files.
OK, lets do that, but keep a comment here saying that those come from travis via (what?) plugin

Comment (patriknw) [inline project/Common.scala]: I see now that version is from `++$TRAVIS_SCALA_VERSION` in the travis script?
how does that work if I build locally and don't specify any scala version?

**Comment by ktoso** [comment]
LGTM, merge at will - would like to skim the docs in preview before we push the release :)

File: README.md [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
-This repository contains interfaces to interact with an Akka Cluster.
\ No newline at end of file
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/akka/akka?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+This repository contains interfaces to interact with an Akka Cluster.
+
+Documentation
+-------------
+
+See [reference](http://developer.lightbend.com/docs/akka-cluster-management/0.1/) and [API](http://developer.lightbend.com/docs/api/akka-cluster-management/0.3/akka/cluster/http/management/index.html) documentation pages.
```
Comment (2m) [inline README.md]: The sentence is "reference and documentation pages ... ". This is copied from Alpakka...

**Comment by 2m** [comment]
Yup. Previewing build docs is very useful for commits which change docs. From time to time I ponder how could we do doc previews for PRs. Have some ideas in mind.

**Comment by 2m** [comment]
@ktoso, here are the docs: http://developer.lightbend.com/docs/akka-cluster-management/0.1-RC1/

**Comment by patriknw** [comment]
nice

**Comment by ktoso** [comment]
Such smooth release process, nice. Colour me impressed.
I really want paradox and docs server to rock :-)

**Pull request merged**